fahrschein/src/main/java/org/zalando/fahrschein/NakadiClient.java

```diff
@@ -54,6 +58,35 @@ public static NakadiClientBuilder builder(URI baseUri) {
         }
     }
 
+    public <T> void publish(String eventName, List<T> events) throws IOException {
+        final URI uri = baseUri.resolve(String.format("/event-types/%s/events", eventName));
+        final ClientHttpRequest request = clientHttpRequestFactory.createRequest(uri, HttpMethod.POST);
+
+        request.getHeaders().setContentType(MediaType.APPLICATION_JSON);
+
+        try (final OutputStream body = request.getBody()) {
+            objectMapper.writeValue(body, events);
```

Comment:
Depends on the settings of the `ClientHttpRequestFactory`, it will either use chunked encoding or buffer into a byte array and then send it including the `Content-Type` header.